dev-tools/reconfigurator-cli/tests/output/cmds-example-stdout

```diff
@@ -1564,13 +1606,22 @@ LEDGERED SLED CONFIG
         b61b7c3c-d665-44b3-9312-794aa81c59de     crucible         install-dataset 
         b957d6cf-f7b2-4bee-9928-c5fde8c59e04     crucible         install-dataset 
         e246f5e3-0650-4afc-860f-ee7114d309c5     crucible         install-dataset 
+    measurements: 
+    install dataset
```

Comment:
What does this mean?  That the current set of measurements that are in-use are coming from the install dataset?

Reply:
Yes, see the comment in https://rfd.shared.oxide.computer/rfd/0512#_measurements_in_reconfigurator about install dataset